extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -394,22 +394,22 @@
 # complex queries responably. Other users can employ thus defined concepts in
 # their queries.
 ##
-$GLOBALS['smwgQConceptCaching'] = CONCEPT_CACHE_HARD; // Which concepts should be displayed only if available from cache?
+$smwDefaults['smwgQConceptCaching'] = CONCEPT_CACHE_HARD; // Which concepts should be displayed only if available from cache?
 		// CONCEPT_CACHE_ALL   -- show concept elements anywhere only if they are cached
 		// CONCEPT_CACHE_HARD  -- show without cache if concept is not harder than permitted inline queries
 		// CONCEPT_CACHE_NONE  -- show all concepts even without any cache
 		// In any cases, caches will always be used if available.
-$GLOBALS['smwgQConceptMaxSize'] = 20; // Same as $smwgQMaxSize, but for concepts
-$GLOBALS['smwgQConceptMaxDepth'] = 8; // Same as $smwgQMaxDepth, but for concepts
+$smwDefaults['smwgQConceptMaxSize'] = 20; // Same as $smwgQMaxSize, but for concepts
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)